modules/base/rendering/renderablenodeline.h

```diff
@@ -50,7 +51,18 @@ class RenderableNodeLine : public Renderable {
     ~RenderableNodeLine() = default;
 
     static documentation::Documentation Documentation();
-    
+
+    // Identifier of the optional distance label
+    std::string _distanceLabelId;
+    //SceneGraphNode* distanceLabelNode = nullptr;
+    //RenderableLabels* distanceLabel;
+
+    // Get the distance between the start and end node
+    double getDistance();
+
+    properties::StringProperty _start;
+    properties::StringProperty _end;
```

Comment:
Consider putting these behind private const getters since we never seem to set them outside this class?

Reply:
Technically they are set through luscripting via the GUI at some point, but this is still available even when they are private (through something we call the propertytree). If this is entirely kosher is debatable, but I will apply your suggestions with some getStart and getEnd functions and call these from 'renderabledistancelabel' instead.